### PR TITLE
Leaf takeover

### PR DESCRIPTION
**Pull request by Spottedleaf**
Finally, my brothers have formed an army and done what was
right. The oppression ends today.

Your blocks are forfeit.

### CODE REVIEW

File: Spigot-Server-Patches/0456-Leaf-takeover.patch [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
++        try {
+         ChunkPacketInfo<IBlockData> chunkPacketInfo = chunk.world.chunkPacketBlockController.getChunkPacketInfo(this, chunk, i, forceLoad); // Paper - Anti-Xray - Add chunk packet info
+         ChunkCoordIntPair chunkcoordintpair = chunk.getPos();
+ 
+@@ -75,7 +77,7 @@ public class PacketPlayOutMapChunk implements Packet<PacketListenerPlayOut> {
+         iterator = chunk.getTileEntities().entrySet().iterator();
+         int totalSigns = 0; // Paper
+ 
+-        while (iterator.hasNext()) {
++        while (false && iterator.hasNext()) { // No such thing as a TE. Only leaves.
```
Comment (zachbr) [inline Spigot-Server-Patches/0456-Leaf-takeover.patch:75]: This is already the best thing ever

**Review by perseus5 — APPROVED** [review]

**Review by Draycia — APPROVED** [review]
I for one accept our new leaf overlords

**Review by EbonJaeger — APPROVED** [review]
Best update in Minecraft history.

**Review by Redned235 — APPROVED** [review]

**Review by DoNotSpamPls — APPROVED** [review]

**Comment by Draycia** [comment]
We may have lost the battle, but we have not lost the war.

**Pull request closed without merging**